src/gamecontroller.cs

```diff
@@ -4,16 +4,18 @@
 // Authors:
 //   Aaron Bockover (abock@xamarin.com)
 //   TJ Lambert (t-anlamb@microsoft.com)
+//   Whitney Schmidt (whschm@microsoft.com)
 //
 // Copyright 2013, 2015 Xamarin Inc.
-// Copyright 2019 Microsoft Corporation
+// Copyright 2019, 2020 Microsoft Corporation
 
 using System;
 
 using CoreFoundation;
 using Foundation;
 using ObjCRuntime;
 using OpenTK;
+//#using CoreHaptics; // need CoreHaptics update for TV 14.0 inclusion
```

Comment:
PR with support for CoreHaptics on tvOS landed on the xcode12 branch: https://github.com/xamarin/xamarin-macios/pull/9273